lesson07/assignment/src/App.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <div id="app">
+    <nav>
+    <img
+      src="https://scontent-sea1-1.xx.fbcdn.net/v/t31.0-8/p960x960/12006472_1049073615127428_5637519699861230041_o.jpg?_nc_cat=110&_nc_oc=AQnEerYvZdDFwM---fIDh8opxiS3cVH1Qt4ssJPb_OYSVx4WVrxDctJa7dIQd-2vwxVmGpULGs_sO88ZogB5hhI_&_nc_ht=scontent-sea1-1.xx&oh=b17190c371e0a77d0e9201f11fc12399&oe=5EA91EAD"
+      alt="nav-pic">
+    <router-link to="/" class="text-primary">Home</router-link>
+    <router-link to="/Images" class="text-success">Images</router-link>
+    <router-link to="/Accordion" class="text-danger">Accordion</router-link>
+    <router-link to="/Grid" class="text-warning">Grid</router-link>
+  </nav>
+  <b-container>
+    <router-view/>
+  </b-container>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'app',
+  data () {
+    return {
+
+    }
+  }
+}
+
+</script>
+
+<style lang="scss">
+@import url('https://fonts.googleapis.com/css?family=Muli&display=swap');
+
+#app {
+  // global style
+  font-family: 'Muli', sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  box-sizing: border-box;
+  margin-left: 160px;
+}
+
+nav {
+  height: 100%;
+  /* Full-height: remove this if you want "auto" height */
+  width: 160px;
```

Comment:
Sizing should always be done with `rem` or `%`. There are very few exceptions, and this is a cornerstone of responsive design.